MyHeartCounts/Onboarding/FinalEnrollmentStep.swift

```diff
@@ -62,7 +64,21 @@ struct FinalEnrollmentStep: View {
             return
         }
         do {
-            try await studyManager.enroll(in: study)
+//            if let enrollmentDate = account.details?.dateOfEnrollment {
```

Comment:
Todo not here to re-enable this once we have the fix by @PaulGoldschmidt in place 👍